Package.swift

```diff
@@ -4,7 +4,7 @@ import PackageDescription
 let package = Package(
     name: "ManagedAppConfigLib",
     platforms: [
-        .macOS(.v10_15),
+        .macOS(.v11),
```

Comment:
Is this not a breaking change?

Reply:
Managed App Config does not work on macOS older than v11.  I'm not sure why this was accepted as targeting `.v10_15`.